pkg/addons/data/ConfigMap/openshift-node/node-config-compute.yaml

```diff
@@ -21,42 +27,40 @@ data:
       - /etc/origin/node/bootstrap.kubeconfig
       cert-dir:
       - /etc/origin/node/certificates
-      cloud-config:
-      - /etc/origin/cloudprovider/azure.conf
-      cloud-provider:
-      - azure
+      container-runtime:
+      - remote
+      container-runtime-endpoint:
+      - /var/run/crio/crio.sock
```

Comment:
This moves us to crio as our runtime.  I know this has been tested as I led this effort in online.  Just make a note to tell everyone that this is coming.  We should maybe have a brief explanation of crio tooling.

Reply:
Nice :) we still have docker installed. but goes would be to remove it with time. But I think its time. It will make transition to 4.0 easier.